addon/styles/addon.scss

```diff
@@ -74,7 +74,7 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
       margin-left: 10px;
     }
 
-    &-text {
```

Comment:
I believe we've technically said breaking CSS classes requires a MAJOR change in case a consumer is using custom CSS. In this case I don't think anyone is and would be okay letting this slide in a #MINOR# change. @sglanzer thoughts? I figure we can let it slide or add both classes to the component for now with a TODO comment to remove `-text` in the next major.